sql/moz-fx-data-shared-prod/fenix_derived/attributable_clients_v1/query.sql

```diff
@@ -0,0 +1,106 @@
+CREATE TEMP FUNCTION sum_map_values(map ARRAY<STRUCT<key STRING, value INT64>>)
+RETURNS INT64 AS (
+  (SELECT SUM(value) FROM UNNEST(map))
+);
+
+WITH client_day AS (
+  SELECT
+    DATE(submission_timestamp) AS submission_date,
+    sample_id,
+    client_info.client_id,
+    LOGICAL_OR(
+      mozfun.norm.extract_version(client_info.app_display_version, 'major') >= 107
+    ) AS has_search_data,
+    SUM(sum_map_values(metrics.labeled_counter.browser_search_with_ads)) AS searches_with_ads,
+    SUM(sum_map_values(metrics.labeled_counter.browser_search_ad_clicks)) AS ad_clicks,
+  FROM
+    mozdata.fenix.baseline
+  WHERE
+    DATE(submission_timestamp) = @submission_date
+  GROUP BY
+    submission_date,
+    sample_id,
+    client_id
+),
+searches AS (
+  SELECT
+    submission_date,
+    client_id,
+    SUM(search_with_ads) AS searches_with_ads,
```

Comment:
Add search counts here. Search with ads can be unreliable so better to have both